src/apps/common/submodules/navigationWizard/navigationWizard.scss

```diff
@@ -0,0 +1,104 @@
+@import "../../../../css/partials/base";
+
+.navigation-wizard .header {
```

Comment:
An `id` should be used instead of a `class` to identify the common control containers.

You can use something like `navigation_wizard_wrapper` or `navigation_wizard_common` to have a more unique id.